PowerEditor/src/WinControls/Preference/preferenceDlg.cpp

```diff
@@ -4982,9 +4984,14 @@ intptr_t CALLBACK SearchingSubDlg::run_dlgProc(UINT message, WPARAM wParam, LPAR
 		{
 			switch (wParam)
 			{
-				case IDC_CHECK_STOPFILLINGFINDFIELD:
+				case IDC_CHECK_FILL_FIND_FIELD_WITH_SELECTED:
 				{
-					nppGUI._stopFillingFindField = isCheckedOrNot(IDC_CHECK_STOPFILLINGFINDFIELD);
+					nppGUI._fillFindFieldWithSelected = isCheckedOrNot(IDC_CHECK_FILL_FIND_FIELD_WITH_SELECTED);
+					::EnableWindow(::GetDlgItem(_hSelf, IDC_CHECK_FILL_FIND_FIELD_SELECT_CARET), nppGUI._fillFindFieldWithSelected ? TRUE :FALSE);
+					if (!nppGUI._fillFindFieldWithSelected) {

```

Comment:
Please respect coding standard:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#general

Reply:
Sorry about that...